### DISCUSSION TITLE
memes are keep repeating themeselves

### DISCUSSION THREAD

**Discussion by thespecialone1** [halitsever/random-memes#10] [General]
I'm using this api in my whatsapp bot it sends quality memes but only few, after 5-7 memes, the memes starts to repeat themselves.
Can you fix that?

**Comment by halitsever** [comment]
I'm working on this, there will be more meme content in the next release.

**Comment by halitsever** [comment]
I made a new api service. It brings more quality and different content through Reddit.

https://github.com/halitsever/random-memes/issues/11
